### PR TITLE
Add MaterialData for DoublePlant.

### PR DESCRIPTION
**Pull request by jimmikaelkael**
This will be necessary as per this PR on Glowstone:
https://github.com/GlowstoneMC/Glowstone/pull/374

---

_Edited by turt2live_

**Related links:**
- Glowstone: None needed

### CODE REVIEW

**Comment by turt2live** [comment]
@jimmikaelkael This PR is not mergeable. Please resolve the issue as soon as possible, thanks!

Edit: And I'm going to skip formatting review on this, look at your other material data PRs for formatting concerns.

**Comment by jimmikaelkael** [comment]
@turt2live tried to follow your advices given of the other PRs but still may not be perfect.

File: src/main/java/org/bukkit/material/DoublePlant.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    public DoublePlant(final int type) {
+        super(type);
+    }
+
+    public DoublePlant(final Material type) {
+        super(type);
+    }
+
+    /**
+     *
```
Comment (turt2live) [inline src/main/java/org/bukkit/material/DoublePlant.java]: Extra line here (and above similar javadocs)

File: src/main/java/org/bukkit/material/DoublePlant.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+     */
+    @Deprecated
+    public DoublePlant(final Material type, final byte data) {
+        super(type, data);
+    }
+
+    /**
+     * Gets the current species of this double plant.
+     *
+     * @return DoublePlantSpecies of this double plant
```
Comment (turt2live) [inline src/main/java/org/bukkit/material/DoublePlant.java]: `@return The ...` (missing 'The')

File: src/main/java/org/bukkit/material/DoublePlant.java [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+     */
+    public DoublePlantSpecies getSpecies() {
+        return DoublePlantSpecies.getByData(getData());
+    }
+
+    /**
+     * Sets the species of this double plant.
+     *
+     * @param species New species of this double plant
```
Comment (turt2live) [inline src/main/java/org/bukkit/material/DoublePlant.java]: `@return The ...` (missing 'The')

**Comment by turt2live** [comment]
@jimmikaelkael Just missed a few things. Please fix them when you get the chance :3

**Comment by jimmikaelkael** [comment]
@turt2live Yeah :) Should be fixed.

**Comment by SpaceManiac** [comment]
Merged in 267a795149a14250209b822ec99804c335150a56.

**Pull request closed without merging**